cmd/speedle-ads/main.go

```diff
@@ -69,18 +73,54 @@ func main() {
 	evaluator, err := newEvaluator(conf)
 	if err != nil {
 		log.Panic(err)
-		return
+		os.Exit(1)
```

Comment:
log.Panic() will call panic(), which will exit with a non-zero status.    
BTW, will we have different exit code for different error?

Reply:
Done, use log.Fatal() instead.